python/django/string-field-null-checks.yml

```diff
@@ -0,0 +1,28 @@
+rules:
+  - id: no-null-string-field
+    patterns:
```

Comment:
Do you want to add `pattern-inside` and assignment checks here?

Reply:
👍